Kernel/Sections.h

```diff
@@ -16,7 +16,8 @@
 #define UNMAP_AFTER_INIT NEVER_INLINE __attribute__((section(".unmap_after_init")))
 
 #define KERNEL_PD_END (kernel_mapping_base + KERNEL_PD_SIZE)
-#define KERNEL_PT1024_BASE (kernel_mapping_base + 0x3FE00000)
+#define KERNEL_PT1024_OFFSET 0x3FE00000
+#define KERNEL_PT1024_BASE (kernel_mapping_base + KERNEL_PT1024_OFFSET)
```

Comment:
:thinking:  are these sections here aarch64-exclusive? If so, should we wrap them in an `#if ARCH(aarch64)`  ?

Reply:
No these are definitely not aarch64-specific, and are used in the MemoryManager for getting the quickmapping page. x86_64 should use this macro when setting up the page tables, but currently it doesn't use it.